src/darwin/Framework/CHIP/MTRBaseDevice_Internal.h

```diff
@@ -27,6 +27,14 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
+static inline MTRTransportType MTRMakeTransportType(chip::Transport::Type type) {
```

Comment:
Why does this need to be in the header?  Can't it just go in MTRDeviceController.mm?

Reply:
It's effectively just a static_cast so I think it makes sense to define it as `inline` ?